server/chat-commands/room-settings.ts

```diff
@@ -880,6 +880,38 @@ export const commands: Chat.ChatCommands = {
 		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a public room you have staff in.`,
 		`Only users who are staff in a public room or global auth can make groupchats.`,
 	],
+	battlechat(target, room, user) {
+		room = this.requireRoom();
+		this.checkChat();
+		if (!room.battle) return this.errorReply("/battlechat must be used in a battle.");
+		const roomid = room.roomid.replace('battle-', 'battlechat-') as RoomID;
+		if (Rooms.get(roomid)) {
+			if (!this.runBroadcast()) return;
+			this.sendReplyBox(`Join the battle chat: <a href="/${roomid}">${roomid}</a>`);
+			return;
+		}
+		if (!user.can('mute', null, room)) {
+			this.sendReply("This battle does not have a side chat.");
+			return;
+		}
+		const targetRoom = Rooms.createChatRoom(roomid, "[BC] " + room.title, {
+			// TODO: come up with a good titling scheme
```

Comment:
`Chat for ${room.title}` would probably be fine; for example, "Chat for Player A vs. Player B".